### DISCUSSION TITLE
foo.Containerfile not recognised

### DISCUSSION THREAD

**Discussion by ananthb** [github-linguist/linguist#6465] [Classification]
foo.Containerfile is not recognised as a Dockerfile/Containerfile.

**Comment by lildude** [comment]
You want to open a [feature request](https://github.com/github-linguist/linguist/issues/new?assignees=&labels=Improvement&projects=&template=feature_request.md&title=) for that or submit a PR to implement it yourself, assuming it meets the usage requirements.
